ci/cron/compare-perf.sh

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+BASELINE=$1
+
+measure() {
+    local treeish=$1
+    local out=$(mktemp -d)/out.json
```

Comment:
Looks like we are never deleting this directory?

Reply:
Indeed. The job runs once a day and the one file that function writes is about 8.8kb, so I don't think it's an issue. Even taking into account the fact that the script invokes that function twice, and FS padding (on HFS+ at least), this is still consuming only 24kb in total (as machines get cleared every day, there should never be more than one set of files).